VARIABLES.sh

```diff
@@ -20,6 +20,11 @@ MountPointSystemExt="/mnt/system_ext"
 MountPointSystem="/mnt/system"
 MountPointVendor="/mnt/vendor"
 
+# Conditional System.img size
+SystemImageSize=$(du "$ImagesRoot"/system.img | tr "  " "*" | tr "\t" " " | cut -d " " -f 1)
+OpenGappszipSize=$(du "$GAppsRoot"/*.zip | tr "  " "*" | tr "\t" " " | cut -d " " -f 1)
+FinalSize=$(((SystemImageSize+OpenGappszipSize+1000000000)/1024))
```

Comment:
Are you sure about this?
`(SystemImageSize+OpenGappszipSize+1000000000)/1024`
It generates a very big number in my machine
how about `((SystemImageSize+OpenGappszipSize)/1024)+128` or 256?

Reply:
or removing 3 zeros is fine to make sure the script won't fail when the compresses ratio is high

Reply:
I'll research back on OpenGappps zip to find out the approx max storage they occupy when totally extracted

Reply:
Ok it's around ~62% compression ration
![image](https://user-images.githubusercontent.com/68287637/172054583-5b2eb973-e9c4-4ec1-9b4b-ed85b138d25b.png)
X3 times is better than hardcoded value 1000000000

> thanks for noticing me @abaynp